apps/rush-lib/src/cli/taskRunner/ProjectTask.ts

```diff
@@ -31,18 +31,32 @@ export default class ProjectTask implements ITaskDefinition {
     return this._rushProject.packageName;
   }
 
+  public isIncrementalBuildAllowed: boolean;
+
   private _hasWarningOrError: boolean;
+  private _rushProject: RushConfigurationProject;
+  private _rushConfiguration: RushConfiguration;
+  private _commandToRun: string;
+  private _customFlags: string[];
+  private _ignoreMissingScript: boolean;
   private _packageChangeAnalyzer: PackageChangeAnalyzer;
 
   constructor(
-    private _rushProject: RushConfigurationProject,
-    private _rushConfiguration: RushConfiguration,
-    private _commandToRun: string,
-    private _customFlags: string[],
-    public isIncrementalBuildAllowed: boolean,
-    private _ignoreMissingScript: boolean
+    rushProject: RushConfigurationProject,
+    rushConfiguration: RushConfiguration,
+    commandToRun: string,
+    customFlags: string[],
+    isIncrementalBuildAllowed: boolean,
+    ignoreMissingScript: boolean,
+    packageChangeAnalyzer: PackageChangeAnalyzer
```

Comment:
There's enough stuff here that we should probably move it into an IProjectTaskParameters interface.  Maybe out of scope for this PR